app/src/components/button/button_stateful/index.tsx

```diff
@@ -40,29 +40,37 @@ const IdleDisabledCSS = css`
     opacity: 0.5;
   }
 `
+const SuccessCSS = css`
```

Comment:
I assume there's no way to get these to show without `!important`? generally speaking makes code harder to maintain

Reply:
Yeah in general it isn't good to use important but in this case I think it makes sense since this class is applied conditionally I don't see other way without repeating some code to increase specificity of that class. Do you have some suggestion?